run.sh

```diff
@@ -15,5 +15,21 @@ for pkg in libpigpio-dev libpigpiod-if-dev pigpio; do
     fi
 done
 
+# start the pigpiod service
+echo "Starting pigpiod service..."
+sudo systemctl start pigpiod
+sudo systemctl enable pigpiod
```

Comment:
sudo isn't needed here, you're already running as root. Sudo may also not actually be installed.

Secondly, enable isn't likely useful here, as it enables start-at-boot... doesn't have to be done every time, just after the install section above.